src/cli/cli.ts

```diff
@@ -1,10 +1,10 @@
 import * as fs from 'fs';
 import * as path from 'path';
 import { Command } from 'commander';
-import { replaceSuffix, isValidSolFile, outputResult } from './io';
-import { compileSolFiles } from './solCompile';
-import { handleTranspilationError, transform, transpile } from './transpiler';
-import { analyseSol } from './utils/analyseSol';
+import { replaceSuffix, isValidSolFile, outputResult } from '../io';
```

Comment:
Why do we need to make a new folder for cli?

Reply:
Placing the file "cli.test.ts" in the "tests/" directory would cause the "build" job to run it, , that I don't want